code/modules/research/circuitprinter.dm

```diff
@@ -151,18 +151,17 @@ using metal and glass, it uses glass and reagents (usually sulphuric acid).
 		return
 	if(amount > stack.get_amount())
 		amount = stack.get_amount()
-	if(max_material_storage - TotalMaterials() < (amount * SHEET_MATERIAL_AMOUNT)) //Can't overfill
-		amount = min(stack.get_amount(), round((max_material_storage - TotalMaterials()) / SHEET_MATERIAL_AMOUNT))
+	if(max_material_storage - TotalMaterials() < amount) //Can't overfill
+		amount = min(stack.get_amount(), max_material_storage - TotalMaterials())
 
 	busy = 1
-	use_power(max(1000, (SHEET_MATERIAL_AMOUNT * amount / 10)))
-	var/stacktype = stack.type
-	var/t = getMaterialName(stacktype)
+	use_power(max(1000, amount / 10))
```

Comment:
`(max(1000, amount / 10)` pobably always will return 1000 because there is no material with 100+ unints per stack.